### DISCUSSION TITLE
does argocd upgrade helm charts automatically

### DISCUSSION THREAD

**Discussion by pdeva** [argoproj/argo-cd#11989] [Q&A] [answered]
if i specify a semver range for a helm chart in an application definition, will argocd automatically upgrade the chart when a newer version of that chart is available?

**Comment by jannfis** [comment, marked as answer]
Yes, but only for Charts from classic (non-OCI) repositories. Please refer to https://argo-cd.readthedocs.io/en/stable/user-guide/tracking_strategies/#helm for further information.
